.circleci/config.yml

```diff
@@ -0,0 +1,125 @@
+defaults: &defaults
+  working_directory: ~/SalesforceMobileSDK-ReactNative
+  macos:
+    xcode: "9.4.1"
+  shell: /bin/bash --login -eo pipefail
+  environment:
+    BASH_ENV: ~/.bashrc
+    FASTLANE_SKIP_UPDATE_CHECK: true
+    CHRUBY_VER: 2.5.1
+
+version: 2
+jobs:
+  test-ios:
+    <<: *defaults
+    steps:
+      - checkout
+
+#      - restore_cache:
+#          key: v5-gems-{{ checksum ".circleci/Gemfile.lock" }}
```

Comment:
Should this commented out section go away?

Reply:
This is still work in progress.
When the PR is ready to merge, those commented sections should either be uncommented (and working) or gone.